cors.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "origin": ["http://localhost:3000", "https://hoverboard-v2-dev.firebaseapp.com/"],
```

Comment:
yes, ideally users should be able to have a functional website after modifying `hoverboard.config.json`